src/plugins/dashboard/public/dashboard_actions/unlink_from_library_action.tsx

```diff
@@ -55,13 +55,7 @@ export class UnlinkFromLibraryAction implements Action<EmbeddableApiContext> {
   public readonly id = ACTION_UNLINK_FROM_LIBRARY;
   public order = 15;
 
-  private toastsService;
-
-  constructor() {
-    ({
-      notifications: { toasts: this.toastsService },
-    } = pluginServices.getServices());
-  }
+  constructor() {}
```

Comment:
Constructor can be removed

Reply:
Removed in b37dfaadac70cdb1ddc7bfb0b6cad3d5c8a42ebf